### PR TITLE
docs: change README

### PR DESCRIPTION
**Pull request by amandahla**
Applicable spec: <link>

### Overview

This PR changes README default file.

- Add introduction
- Add architecture diagram
- Fix links

### Rationale

Add README to the charm repo.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [ ] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

No charmhub available yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,20 +7,42 @@ Avoid using this README file for information that is maintained or published els
...
+## Project and community
+
+The Maubot Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+* [Contribute](https://charmhub.io/maubot/docs/contributing)
+* [Getting Started](https://charmhub.io/maubot/docs/getting-started)
```
Comment (erinecon) [inline README.md:47]: This link didn't lead me anywhere. Is the problem on my end, or will this link eventually work?

Comment (amandahla) [reply README.md:49]: You are fast :-D
Since this charm is a WIP yet, I didn't publish anything so the charmhub page is only registered, not listed.

Comment (erinecon) [reply README.md:49]: I got excited when I saw a README PR for a charm I didn't recognize :sweat_smile: 

Sounds good about the Charmhub page, as long as the links will be up and running soon then everything looks good to me! :+1:

Comment (amandahla) [reply README.md:49]: I will check after publishing a charm version but usually is expected to be working once one is available

File: README.md [hunk 2]
```diff
@@ -7,20 +7,44 @@ Avoid using this README file for information that is maintained or published els
...
+Maubot is a plugin-based [Matrix](https://matrix.org/) bot system written in Python.
 
-## Other resources
+## Architecture
 
-<!-- If your charm is documented somewhere else other than Charmhub, provide a link separately. -->
+The charm has the following architecture:
 
-- [Read more](https://example.com)
+```mermaid
```
Comment (yanksyoon) [inline README.md:25]: Is this supported on discourse? (That is awesome if it is!)

**Review by yanksyoon — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (amandahla) [reply README.md:25]: Good question, I know that there is this theme
https://meta.discourse.org/t/discourse-mermaid/218242
But I'm not sure if is enabled

File: .trivyignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Pebble CVE
```
Comment (merkata) [inline .trivyignore:1]: Oh what's that, can you add a bit of context - severity and a couple of words on what it is about?

File: README.md (hunk 2 above)
Comment (merkata) [reply README.md:25]: Not sure of the syntax, though since Discourse v3 we support Mermaid.

**Review by merkata — APPROVED** [review]

**Review by Thanhphan1147 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (amandahla) [reply README.md:25]: Great :)

File: .trivyignore (hunk 3 above)
Comment (amandahla) [reply .trivyignore:1]: I'll add it in the #6 then

**Pull request merged**